xonsh/color_tools.py

```diff
@@ -479,3 +480,16 @@ def make_palette(strings):
             t, _, s = t.partition(" ")
             palette[t] = rgb_to_ints(t)
     return palette
+
+
+def warn_deprecated_no_color():
+    """Show a warning once if NO_COLOR was used instead of RESET."""
+    global _NO_COLOR_WARNING_SHOWN
+    if not _NO_COLOR_WARNING_SHOWN:
+        import warnings
+
+        warnings.warn(
```

Comment:
This should probably just be a print to stderr

Reply:
Fixed in ab97137